archetype/config/database.js

```diff
@@ -1,37 +0,0 @@
-/**
```

Comment:
we should provide an archetype for stores and model now right ?

Reply:
I think that would be a good idea.  I'll add that to this PR.